### DISCUSSION TITLE
It is possible hide navigation for previous/next section/page?

### DISCUSSION THREAD

**Discussion by michalzobec** [squidfunk/mkdocs-material#3489] [Q&A] [answered]
Hello,

thank you for your work, I working on documentation portal, and your web generator template is great. I have little problem with your navigation for previous/next section or page ... it is possible hide it? I want completelly hide/off this feature.

I do not know if name "section" is correct, I mean navigation at down of page where you can go to next section or page, or back.

for example here in your documentation https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/

![2022-01-22 22_35_16-Window](https://user-images.githubusercontent.com/13049276/150656252-a9493ab9-2786-45dc-bf92-a83cdd6bd04f.png)  ![2022-01-22 22_35_33-Window](https://user-images.githubusercontent.com/13049276/150656258-58117fe6-20fd-4490-8e4c-77deff75ea02.png)

thank you for your help.

**Comment by squidfunk** [comment, marked as answer]
Yes, of course! The easiest way is to use [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css):

``` css
.md-footer__inner {
  display: none;
}
```

**Reply by michalzobec** [reply]
hello, thank you, but it is not help ...

I use version mkdocs-material-7.0.0b2.

URL is: https://pomoc2.zobecdev.cz/microsoft-365-43a5/

**Reply by squidfunk** [reply]
In that case:

``` css
.md-footer-nav {
  display: none;
}

**Reply by michalzobec** [reply]
`
.md-footer-nav { 
  display: none;
}
`

yes! this it work! :) thank you very much :)

**Comment by chancsy** [comment]
With the release of 8.3.0 which added the support for per-page footer navigation hiding, it seems this has deprecated, is there a new way to hide the footer navigation globally? I am still new to CSS so can't seem to find a way myself.

**Reply by squidfunk** [reply]
You should still use CSS or theme extension to hide the footer navigation on _all_ pages. We might introduce a new feature flag that will do this for you, but this will be bound to a new major release which we just won't do at this time, as it's not necessary.

**Reply by chancsy** [reply]
Thanks for the reply, I managed to find the updated CSS class to override it with.
The original class was changed to facilitate the new hidden flag, so adding this in my `extra.css` did the trick.
```
.md-footer__inner:not([hidden]) {
    display: none
}
```

**Reply by squidfunk** [reply]
Thanks for sharing!

**Reply by McDic** [reply]
This isn't working in `mkdocs==1.5.3` and `mkdocs-material==9.5.3`. I don't know much about front-end stuffs but seems `docs/stylesheets/extra.css` is not overriding built-in components. What should I do?

**Reply by kamilkrzyskow** [reply]
Hello @McDic,
since version 9.0.0 of `mkdocs-material` the back and next page buttons are disabled by default, so CSS shouldn't be necessary:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#navigation

What do you want to achieve? Also just to make sure the `extra_css` path shouldn't include the actual `docs/` root directory.
https://squidfunk.github.io/mkdocs-material/customization/#additional-css

**Reply by McDic** [reply]
Oh! I am using built-in blog plugin so that plugin is probably enabling the back and next page buttons by default. I wanted to manipulate "next page" and "prev page" buttons for any of followings;

- Disable it
- Make it navigate to the previous/next article in same category

At least now I can disable it by including `extra_css` in `mkdocs.yml`, that was my fault.

Thank for your team for providing very nice blogging tool for lazy people like me. `mkdocs` and `mkdocs-materials` are very helpful!!

**Reply by kamilkrzyskow** [reply]
You added the `navigation.footer` prev and next buttons 4 hours ago https://github.com/McDic/BlogV2/commit/7c2c5d0dfeea4b6f19a8eafdb8faae1516985ee8 @McDic. I don't think that blog changes it in any way. 

> Disable it

They should be disabled by default without `navigation.footer` as linked above. 

> Make it navigate to the previous/next article in same category

Changing to what the `prev` and `next` buttons point to and limiting them to operate in categories only would be difficult. 
You would have to create a hook and patch the internals of the blog plugin. To be honest I'm not sure how well it would work out 🤔 
https://www.mkdocs.org/user-guide/configuration/#hooks
https://github.com/squidfunk/mkdocs-material/blob/246266fafad2c2fdc78022c14f7f4b2df91167f3/src/plugins/blog/plugin.py#L622-L624

**Reply by McDic** [reply]
I see. I think I added `navigation.footer` because I wanted to include social links and copyright notice at the bottom of the blog. I guess internal plugin hack would be a difficult stuff for me now, but when I have some chance I will look at it. Thanks!

**Reply by McDic** [reply]
I did this!

- Code: https://github.com/McDic/BlogV2/tree/master/custom_plugin
- Demo (Main content is in Korean but you can still see prev/next buttons): https://blog.mcdic.net/tho-3/
